plugins/module_utils/k8s/resource.py

```diff
@@ -0,0 +1,126 @@
+import os
```

Comment:
Can we add a license boilerplate here? and a changelog entry for this change.

Reply:
License definitely. My thinking is to create a single changelog entry for the refactor when we merge this branch into main.